packages/server/src/course/heatmap.service.ts

```diff
@@ -228,6 +242,6 @@ export class HeatmapService {
     autoExit: true,
   })
   async create(): Promise<void> {
-    const h = await this.getHeatmapFor(4);
+    const h = await this._getHeatmapFor(4);
```

Comment:
This wasn't from this PR but why is `4` is hardcoded in the `heatmap:generate` command?

Reply:
^^^^ it seems to be a dev thing but... still concerning

Reply:
I think this was a command thing, in a separate PR I'm gonna update this command